Makefile

```diff
@@ -7,6 +7,7 @@ LOCAL_BRIDGE_PLUGIN_BINARY=bin/plugins/ecs-bridge
 GIT_PORCELAIN=$(shell git status --porcelain | wc -l)
 GIT_SHORT_HASH=$(shell git rev-parse --short HEAD)
 VERSION=$(shell cat $(ROOT)/VERSION)
+GO_EXECUTABLE=$(shell which go)
```

Comment:
Minor, but `which` is actually not guaranteed to be present. It's not, for example, on the Amazon Linux base docker image, and it's not required by POSIX. We should use `command -v`, which is required by POSIX and thus more compatible. Alternatively, document a dependency on `/bin/sh` being `bash` so we can make use of the built-in.